packages/rum-core/src/domain/contexts/pageStateHistory.ts

```diff
@@ -1,7 +1,18 @@
 import type { Duration, RelativeTime } from '@datadog/browser-core'
-import { addDuration, addEventListeners, relativeNow, DOM_EVENT } from '@datadog/browser-core'
+import {
+  elapsed,
+  ValueHistory,
+  SESSION_TIME_OUT_DELAY,
+  toServerDuration,
+  addEventListeners,
+  relativeNow,
+  DOM_EVENT,
+} from '@datadog/browser-core'
+import type { PageStateServerEntry } from '../../rawRumEvent.types'
 
-export const MAX_PAGE_STATE_ENTRIES = 500
+export const MAX_PAGE_STATE_ENTRIES = 200
```

Comment:
I updated the limits. Basically, I want to base the foreground computation on pageStateHistory.ts as a follow up, so:

- [MAX_PAGE_STATE_ENTRIES](https://github.com/DataDog/browser-sdk/blob/bfdfcb34756934df9a433463921b00ee2138c01e/packages/rum-core/src/domain/contexts/pageStateHistory.ts#L14) > [MAX_NUMBER_OF_STORED_FOREGROUND_PERIODS](https://github.com/DataDog/browser-sdk/blob/bfdfcb34756934df9a433463921b00ee2138c01e/packages/rum-core/src/domain/contexts/foregroundContexts.ts#L8) because we will have more states
- [MAX_PAGE_STATE_ENTRIES_SELECTABLE](https://github.com/DataDog/browser-sdk/blob/bfdfcb34756934df9a433463921b00ee2138c01e/packages/rum-core/src/domain/contexts/pageStateHistory.ts#L16) === [MAX_NUMBER_OF_SELECTABLE_FOREGROUND_PERIODS](https://github.com/DataDog/browser-sdk/blob/bfdfcb34756934df9a433463921b00ee2138c01e/packages/rum-core/src/domain/contexts/foregroundContexts.ts#L6) to be consistent